lighthouse-core/gather/gather-runner.js

```diff
@@ -165,6 +166,20 @@ class GatherRunner {
     } else if (mainRecord.hasErrorStatusCode()) {
       errorDef = {...LHError.errors.ERRORED_DOCUMENT_REQUEST};
       errorDef.message += ` Status code: ${mainRecord.statusCode}.`;
+    } else if (!mainRecord.finished) {
+      // could be security error
+      const securityState = await driver.getSecurityState();
```

Comment:
Small nit but I think it'd be slightly better to collect this on L288 and pass that into getPageLoadError() rather than all of driver.